src/implementations/roaring.rs

```diff
@@ -67,67 +66,44 @@ mod tests {
 	}
 
 	#[test]
-	fn test_roaring_serialization_benchmark() {
+	fn test_roaring_serialization_space_benchmark() {
```

Comment:
If this is still a benchmark do we still need it here since we have it specifically as a benchmark now?

Reply:
I am moving everything to the benchmark